### PR TITLE
Add a description in Japanese.

### PR DESCRIPTION
**Pull request by LibraryLibrarian**
links to the JapaneseWe have added a `README.ja-JP.md` translated into Japanese.
Links to the Japanese description have been added to both `README.md` and `README.zh-CN.md`.

### CODE REVIEW

File: README.ja-JP.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- [ ] 目次
+
+## 🚀 使用方法
+
+1. [テンプレート](https://github.com/saicaca/fuwari/generate)から新しいリポジトリを作成するかCloneをします。
+2. ブログをローカルで編集するには、リポジトリをクローンした後、`pnpm install` と `pnpm add sharp` を実行して依存関係をインストールします。  
+   - [pnpm](https://pnpm.io)がインストールされていない場合は `npm install -g pnpm` で導入可能です。
+3. `src/config.ts`ファイルを編集する事でブログを自分好みにカスタマイズします。
+4. `pnpm new-post <filename>`で新しい記事を作成して、`src/content/posts/`.フォルダ内で編集します。
+5. 作成したブログをVercel、Netlify、GitHub Pagesなどにデプロイするには[ガイド](https://docs.astro.build/en/guides/deploy/)に従って下さい。
```
Comment (saicaca) [inline README.ja-JP.md:27]: This sentence in the EN version was just updated, could you please update the translation?

And replacing the document link with the JP version(https://docs.astro.build/ja/guides/deploy/) would be better.

File: README.ja-JP.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# 🍥Fuwari
+
+[Astro](https://astro.build)で構築された静的ブログテンプレート
+
+[**🖥️Live Demo (Vercel)**](https://fuwari.vercel.app)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏English README**](https://github.com/saicaca/fuwari)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏中文 README**](https://github.com/saicaca/fuwari/blob/main/README.zh-CN.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**📦Old Hexo Version**](https://github.com/saicaca/hexo-theme-vivia)
```
Comment (saicaca) [inline README.ja-JP.md:5]: Could you please also translate the link texts here?

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 A static blog template built with [Astro](https://astro.build).
 
-[**🖥️Live Demo (Vercel)**](https://fuwari.vercel.app)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏中文 README**](https://github.com/saicaca/fuwari/blob/main/README.zh-CN.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**📦Old Hexo Version**](https://github.com/saicaca/hexo-theme-vivia)
+[**🖥️Live Demo (Vercel)**](https://fuwari.vercel.app)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏中文 README**](https://github.com/saicaca/fuwari/blob/main/README.zh-CN.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏日本語 README**](https://github.com/saicaca/fuwari/blob/main/README.ja-JP.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**📦Old Hexo Version**](https://github.com/saicaca/hexo-theme-vivia)
```
Comment (saicaca) [inline README.md:5]: The link texts look too long. Maybe this is better:

[**🖥️Live Demo (Vercel)**](https://fuwari.vercel.app)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏中文**](https://github.com/saicaca/fuwari/blob/main/README.zh-CN.md)&nbsp;&nbsp;[**🌏日本語**](https://github.com/saicaca/fuwari/blob/main/README.ja-JP.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**📦Old Hexo Version**](https://github.com/saicaca/hexo-theme-vivia)

**Comment by LibraryLibrarian** [comment]
Sorry, I accidentally edited `README.md`, so I reverted it.

**Comment by LibraryLibrarian** [comment]
I've corrected the parts that received reviews. Regarding the Japanese translation, some English words are known as "[Wasei-eigo](https://en.wikipedia.org/wiki/Wasei-eigo)" (Japanese-made English), and there are cases where their pronunciation in Japanese is exactly the same as in English. (For example, "Version" is expressed as "バージョン" in Japanese but pronounced just like in English.)
In such cases, it might be more readable to leave them in English rather than translating them into Japanese. I have translated everything except for proper nouns like Hexo and Vercel into Japanese for what I've just pushed, but personally, I feel the following form might be better.
[**🖥️Live Demo (Vercel)**](https://fuwari.vercel.app)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏中文**](https://github.com/saicaca/fuwari/blob/main/README.zh-CN.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**🌏日本語**](https://github.com/saicaca/fuwari/blob/main/README.ja-JP.md)&nbsp;&nbsp;&nbsp;/&nbsp;&nbsp;&nbsp;[**📦旧 Hexo Version**](https://github.com/saicaca/hexo-theme-vivia)

Other Japanese people might prefer different expressions. This varies from person to person.

**Comment by saicaca** [comment]
> I've corrected the parts that received reviews. Regarding the Japanese translation, some English words are known as "[Wasei-eigo](https://en.wikipedia.org/wiki/Wasei-eigo)" (Japanese-made English), and there are cases where their pronunciation in Japanese is exactly the same as in English. (For example, "Version" is expressed as "バージョン" in Japanese but pronounced just like in English.) In such cases, it might be more readable to leave them in English rather than translating them into Japanese. I have translated everything except for proper nouns like Hexo and Vercel into Japanese for what I've just pushed, but personally, I feel the following form might be better. [**🖥️Live Demo (Vercel)**](https://fuwari.vercel.app)   /   [**🌏中文**](https://github.com/saicaca/fuwari/blob/main/README.zh-CN.md)   /   [**🌏日本語**](https://github.com/saicaca/fuwari/blob/main/README.ja-JP.md)   /   [**📦旧 Hexo Version**](https://github.com/saicaca/hexo-theme-vivia)
> 
> Other Japanese people might prefer different expressions. This varies from person to person.

🥲I don't understand Japanese very well, so if you think this translation is better, I think you can change it back. Thank you very much for your effort!

**Comment by LibraryLibrarian** [comment]
Let's leave this translation as it is for now.
If there are other Japanese people interested in this content, they should create a separate issue or pull request.

**Comment by saicaca** [comment]
OK, Thank you for your contribution!

**Pull request merged**